contracts/ApproveAndCallFallBack.sol

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.0;
+
+/// @title Approve And Call Fallback Token Interface
```

Comment:
Shouldn't this be `@notice`?

Reply:
Seems like it should be `@dev` "Explain to a developer any extra details"
`@notice` is for "Explain to an end user what this does"
https://docs.soliditylang.org/en/v0.8.21/natspec-format.html#tags